lib/SIL/SILVerifier.cpp

```diff
@@ -4926,6 +4968,7 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
     
     SILInstructionVisitor::visitSILBasicBlock(BB);
     verifyDebugScopeHoles(BB);
+    verifyLineTable(BB);
```

Comment:
Is there a mechanism for "expensive" verifiers?

Reply:
We did one for the ASTVerifier by checking `SWIFT_DISABLE_AST_VERIFIER` in addition to `NDEBUG`. You could add something like that too, or put it behind an -Xllvm flag (which IIRC won't be available in Release builds at all).

Reply:
You're right. Back to the whiteboard ..

Reply:
Now the algorithm should work.